app/subscriber.py

```diff
@@ -38,14 +38,14 @@ def receipt_to_case(message: Message):
             return
         bucket_name, object_name = message.attributes['bucketId'], message.attributes['objectId']
     except KeyError as e:
-        log.error('Pub/Sub Message missing required attribute(s)', missing_attribute=e.args[0])
+        log.error('Pub/Sub Message missing required attribute', missing_attribute=e.args[0])
```

Comment:
I'll leave this as a single callback for now (following some tweaks to "flatten" the try/except blocks) but it might be worth considering in a future PR.